jadx-core/src/main/java/jadx/api/JadxArgs.java

```diff
@@ -45,7 +45,11 @@
 	private boolean respectBytecodeAccModifiers = false;
 	private boolean exportAsGradleProject = false;
 
-	private boolean isFsCaseSensitive;
+	private boolean fsCaseSensitive;
+
+	private boolean renameCaseSensitive = true;
+	private boolean renameValid = true;
+	private boolean renamePrintable = true;
```

Comment:
Please move `RENAME` enum to this class and make instead three fields one using Set.
Also for this set always use `EnumSet` class instead `HashSet`.